Gemfile

```diff
@@ -45,7 +45,7 @@ gem 'bugsnag-capistrano', '< 2', require: false
 # https://github.com/nahi/httpclient/issues/369
 gem 'httpclient', github: '3scale/httpclient', branch: 'ssl-env-cert'
 gem 'oauth2'
-gem 'k8s-client', '>= 0.10'
+gem 'k8s-ruby'
```

Comment:
There was a dependency conflict with the old library.
https://github.com/k8s-ruby/k8s-ruby is a fork of the original library, that is being maintained.